campaignion_email_to_target/src/Tokens.php

```diff
@@ -25,12 +25,18 @@ public static function submissionTokens(array $tokens, Submission $submission) {
           $messages[] = $message;
         }
       }
-      $t = 'campaignion_email_to_target_messages_token';
-      $text = theme([$t, $t . '_' . $submission->node->nid], [
-        'messages' => $messages,
-        'submission' => $submission,
-      ]);
+      $token = [
```

Comment:
These are theme hooks not tokens. This should be extended with the nid-postfixed hook too, and then be passed to `theme()` in one go.

Also we could check if any of our themes actually overrides this. If not we can just replace the old theme hook without caring for backwards compatibility.

Reply:
This seems to be only used in [campaignion_email_to_target.module file](https://github.com/moreonion/campaignion/blob/bc2af86179e0afefa63481b537e00a5a86c4ac20/campaignion_email_to_target/campaignion_email_to_target.module#L275). I didn't find a theme overriding it - would I rather need to go through the database to check if any of the emails would contain the token?

Reply:
There is no token involved in this particular part of the code. The word is only our interpretation of what the `$t` (for theme hook) meant.

Reply:
Yes I got that, sorry, I missed mentioning that  this was crucial information, thank you!
My answer was more pointed toward the rest of the code not that specific bit. However I figured out that I also had renamed the template in [campaignion_email_to_target.module](https://github.com/moreonion/campaignion/blob/bc2af86179e0afefa63481b537e00a5a86c4ac20/campaignion_email_to_target/campaignion_email_to_target.module#L275) which caused the token to not take effect in the thank you email.